src/com/mendix/widget/carousel/components/Carousel.ts

```diff
@@ -1,3 +1,4 @@
+import "../ui/Carousel.css";
```

Comment:
Css below code imports please. Reason for that is that the code imports are more important. Other resources are imported after that (would also include images etc.)